migration/handler.go

```diff
@@ -96,7 +96,11 @@ func (h *upgradeSchemaHandler) Deliver(ctx weave.Context, db weave.KVStore, tx w
 		return nil, errors.Wrap(err, "current schema version")
 	}
 
-	schema := Schema{Pkg: msg.Pkg, Version: ver + 1}
+	schema := Schema{
+		Metadata: &weave.Metadata{Schema: 1},
```

Comment:
pp: this could go into some package level `var InitialVersion = &weave.Metadata{Schema: 1}` so that you don't have to create it everywhere with a "magic number".

Reply:
I agree that this is not great code. I do not know how to solve it yet. I hope that once we will have to write migration a few times we will learn what is painful and what should be improved.

I did not declare it package level because I was afraid that this way we lose track of what number is used where. I think we will learn about this while writing migrations an improve when the time will come.